mage_ai/frontend/components/datasets/Insights/ColumnAnalysis.tsx

```diff
@@ -601,7 +613,51 @@ function ColumnAnalysis({
               />
             </ChartContainer>
           }
-          right={<div />}
+          right={
+            <ChartContainer
+              noPadding={!!correlationsRowData}
+              title="Values with high correlation"
+            >
+              { highCorrelations.length > 0 ? (
+                <SimpleDataTable
+                  columnFlexNumbers={[1, 1, 1]}
+                  columnHeaders={[
+                    {
+                      label: 'Column',
+                    },
+                    {
+                      label: 'Related column',
+                    },
+                    {
+                      label: 'Correlation',
+                    },
+                  ]}
+                  noBorder
+                  rowGroupData={[
+                    {
+                      rowData: highCorrelations,
+                    },
+                  ]}
+                  small
+                />
+              ) : (
```

Comment:
Multi-line ternary operator style is like this:

```
true_or_false
  ? something
  : something
```

Reply:
got it, will update this in #250